ultraplot/config.py

```diff
@@ -405,64 +402,6 @@ def _infer_ultraplot_dict(kw_params):
     return kw_ultraplot
 
 
-def config_inline_backend(fmt=None):
```

Comment:
Why was this function removed?

Reply:
It is moved as a class function now. It was moved there as it was referencing as global var (`rc_ultraplot`). The encapsulation is more appropriate with this recent change.

Reply:
It is here now: https://github.com/Ultraplot/UltraPlot/pull/384/files#diff-849c439d454bf26e1102dce232b08a417344e1d97a4add022d4088cd00b743caR1060

Note this PR looks more heavy than it is. I moved `_int`  and `__init__` to the top of the class to make it read easier. The only major change is moving `rc_ultraplot` and `rc_matplotlib` as class variables and adjusting the logic elsewhere in the code (mainly a couple of references in `figure.py`)

Reply:
To add to this. I looked into to how this function is called, and it also happens inside `Configurator`. I first thought the function was called by `IPython` directly -- but it is not.